packages/vscode-extension/src/debugging/DebugSession.ts

```diff
@@ -98,6 +120,14 @@ export class DebugSession implements Disposable {
     this.session.customRequest("next");
   }
 
+  private async connectCDPDebugger(cdpConfiguration: CDPConfiguration) {
+    await this.session.customRequest("RNIDE_connect_cdp_debugger", cdpConfiguration);
+  }
+
+  public async sendDebugConsoleLog(message: string, source?: DebugSource) {
```

Comment:
"send" doesn't feel right as we are not sending logs. This should rather be "append". Also I believe we should actually report this as "error" and not "log". So maybe this should be `appendDebuggerConsoleEntry` or `writeToDebuggerConsole` and the method should also take the category of the entry (log/warning/error etc)